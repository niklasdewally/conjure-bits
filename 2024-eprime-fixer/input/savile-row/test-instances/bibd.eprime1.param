language ESSENCE' 1.0
letting b be 7
letting v be 7
letting r be 3
letting k be 3
letting  l be 1

