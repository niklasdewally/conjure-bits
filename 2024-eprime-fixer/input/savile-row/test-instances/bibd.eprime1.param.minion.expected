MINION 3

#This file has been automatically generated by TAILOR v0.3.3
#bug-reports: andrea@cs.st-and.ac.uk

**VARIABLES**
BOOL bibd[7,7] 

# auxiliary variables
BOOL aux0 
BOOL aux1 
BOOL aux2 
BOOL aux3 
BOOL aux4 
BOOL aux5 
BOOL aux6 
BOOL aux7 
BOOL aux8 
BOOL aux9 
BOOL aux10 
BOOL aux11 
BOOL aux12 
BOOL aux13 
BOOL aux14 
BOOL aux15 
BOOL aux16 
BOOL aux17 
BOOL aux18 
BOOL aux19 
BOOL aux20 
BOOL aux21 
BOOL aux22 
BOOL aux23 
BOOL aux24 
BOOL aux25 
BOOL aux26 
BOOL aux27 
BOOL aux28 
BOOL aux29 
BOOL aux30 
BOOL aux31 
BOOL aux32 
BOOL aux33 
BOOL aux34 
BOOL aux35 
BOOL aux36 
BOOL aux37 
BOOL aux38 
BOOL aux39 
BOOL aux40 
BOOL aux41 
BOOL aux42 
BOOL aux43 
BOOL aux44 
BOOL aux45 
BOOL aux46 
BOOL aux47 
BOOL aux48 
BOOL aux49 
BOOL aux50 
BOOL aux51 
BOOL aux52 
BOOL aux53 
BOOL aux54 
BOOL aux55 
BOOL aux56 
BOOL aux57 
BOOL aux58 
BOOL aux59 
BOOL aux60 
BOOL aux61 
BOOL aux62 
BOOL aux63 
BOOL aux64 
BOOL aux65 
BOOL aux66 
BOOL aux67 
BOOL aux68 
BOOL aux69 
BOOL aux70 
BOOL aux71 
BOOL aux72 
BOOL aux73 
BOOL aux74 
BOOL aux75 
BOOL aux76 
BOOL aux77 
BOOL aux78 
BOOL aux79 
BOOL aux80 
BOOL aux81 
BOOL aux82 
BOOL aux83 
BOOL aux84 
BOOL aux85 
BOOL aux86 
BOOL aux87 
BOOL aux88 
BOOL aux89 
BOOL aux90 
BOOL aux91 
BOOL aux92 
BOOL aux93 
BOOL aux94 
BOOL aux95 
BOOL aux96 
BOOL aux97 
BOOL aux98 
BOOL aux99 
BOOL aux100 
BOOL aux101 
BOOL aux102 
BOOL aux103 
BOOL aux104 
BOOL aux105 
BOOL aux106 
BOOL aux107 
BOOL aux108 
BOOL aux109 
BOOL aux110 
BOOL aux111 
BOOL aux112 
BOOL aux113 
BOOL aux114 
BOOL aux115 
BOOL aux116 
BOOL aux117 
BOOL aux118 
BOOL aux119 
BOOL aux120 
BOOL aux121 
BOOL aux122 
BOOL aux123 
BOOL aux124 
BOOL aux125 
BOOL aux126 
BOOL aux127 
BOOL aux128 
BOOL aux129 
BOOL aux130 
BOOL aux131 
BOOL aux132 
BOOL aux133 
BOOL aux134 
BOOL aux135 
BOOL aux136 
BOOL aux137 
BOOL aux138 
BOOL aux139 
BOOL aux140 
BOOL aux141 
BOOL aux142 
BOOL aux143 
BOOL aux144 
BOOL aux145 
BOOL aux146 

# aliases

**SEARCH**

PRINT [
bibd]

VARORDER [
bibd,
aux0,aux1,aux2,aux3,aux4,aux5,aux6,aux7,
aux8,aux9,aux10,aux11,aux12,aux13,aux14,aux15,
aux16,aux17,aux18,aux19,aux20,aux21,aux22,aux23,
aux24,aux25,aux26,aux27,aux28,aux29,aux30,aux31,
aux32,aux33,aux34,aux35,aux36,aux37,aux38,aux39,
aux40,aux41,aux42,aux43,aux44,aux45,aux46,aux47,
aux48,aux49,aux50,aux51,aux52,aux53,aux54,aux55,
aux56,aux57,aux58,aux59,aux60,aux61,aux62,aux63,
aux64,aux65,aux66,aux67,aux68,aux69,aux70,aux71,
aux72,aux73,aux74,aux75,aux76,aux77,aux78,aux79,
aux80,aux81,aux82,aux83,aux84,aux85,aux86,aux87,
aux88,aux89,aux90,aux91,aux92,aux93,aux94,aux95,
aux96,aux97,aux98,aux99,aux100,aux101,aux102,aux103,
aux104,aux105,aux106,aux107,aux108,aux109,aux110,aux111,
aux112,aux113,aux114,aux115,aux116,aux117,aux118,aux119,
aux120,aux121,aux122,aux123,aux124,aux125,aux126,aux127,
aux128,aux129,aux130,aux131,aux132,aux133,aux134,aux135,
aux136,aux137,aux138,aux139,aux140,aux141,aux142,aux143,
aux144,aux145,aux146]


**CONSTRAINTS**

lexleq([bibd[_,5]], [bibd[_,6]])
lexleq([bibd[_,4]], [bibd[_,5]])
lexleq([bibd[_,3]], [bibd[_,4]])
lexleq([bibd[_,2]], [bibd[_,3]])
lexleq([bibd[_,1]], [bibd[_,2]])
lexleq([bibd[_,0]], [bibd[_,1]])
lexleq([bibd[5,_]], [bibd[6,_]])
lexleq([bibd[4,_]], [bibd[5,_]])
lexleq([bibd[3,_]], [bibd[4,_]])
lexleq([bibd[2,_]], [bibd[3,_]])
lexleq([bibd[1,_]], [bibd[2,_]])
lexleq([bibd[0,_]], [bibd[1,_]])
product(bibd[0, 0],bibd[1, 0], aux0)
product(bibd[0, 1],bibd[1, 1], aux1)
product(bibd[0, 2],bibd[1, 2], aux2)
product(bibd[0, 3],bibd[1, 3], aux3)
product(bibd[0, 4],bibd[1, 4], aux4)
product(bibd[0, 5],bibd[1, 5], aux5)
product(bibd[0, 6],bibd[1, 6], aux6)
product(bibd[0, 0],bibd[2, 0], aux7)
product(bibd[0, 1],bibd[2, 1], aux8)
product(bibd[0, 2],bibd[2, 2], aux9)
product(bibd[0, 3],bibd[2, 3], aux10)
product(bibd[0, 4],bibd[2, 4], aux11)
product(bibd[0, 5],bibd[2, 5], aux12)
product(bibd[0, 6],bibd[2, 6], aux13)
product(bibd[0, 0],bibd[3, 0], aux14)
product(bibd[0, 1],bibd[3, 1], aux15)
product(bibd[0, 2],bibd[3, 2], aux16)
product(bibd[0, 3],bibd[3, 3], aux17)
product(bibd[0, 4],bibd[3, 4], aux18)
product(bibd[0, 5],bibd[3, 5], aux19)
product(bibd[0, 6],bibd[3, 6], aux20)
product(bibd[0, 0],bibd[4, 0], aux21)
product(bibd[0, 1],bibd[4, 1], aux22)
product(bibd[0, 2],bibd[4, 2], aux23)
product(bibd[0, 3],bibd[4, 3], aux24)
product(bibd[0, 4],bibd[4, 4], aux25)
product(bibd[0, 5],bibd[4, 5], aux26)
product(bibd[0, 6],bibd[4, 6], aux27)
product(bibd[0, 0],bibd[5, 0], aux28)
product(bibd[0, 1],bibd[5, 1], aux29)
product(bibd[0, 2],bibd[5, 2], aux30)
product(bibd[0, 3],bibd[5, 3], aux31)
product(bibd[0, 4],bibd[5, 4], aux32)
product(bibd[0, 5],bibd[5, 5], aux33)
product(bibd[0, 6],bibd[5, 6], aux34)
product(bibd[0, 0],bibd[6, 0], aux35)
product(bibd[0, 1],bibd[6, 1], aux36)
product(bibd[0, 2],bibd[6, 2], aux37)
product(bibd[0, 3],bibd[6, 3], aux38)
product(bibd[0, 4],bibd[6, 4], aux39)
product(bibd[0, 5],bibd[6, 5], aux40)
product(bibd[0, 6],bibd[6, 6], aux41)
watchsumleq([aux35,aux36,aux37,aux38,aux39,aux40,aux41], 1)
watchsumgeq([aux35,aux36,aux37,aux38,aux39,aux40,aux41], 1)
watchsumleq([aux28,aux29,aux30,aux31,aux32,aux33,aux34], 1)
watchsumgeq([aux28,aux29,aux30,aux31,aux32,aux33,aux34], 1)
watchsumleq([aux21,aux22,aux23,aux24,aux25,aux26,aux27], 1)
watchsumgeq([aux21,aux22,aux23,aux24,aux25,aux26,aux27], 1)
watchsumleq([aux14,aux15,aux16,aux17,aux18,aux19,aux20], 1)
watchsumgeq([aux14,aux15,aux16,aux17,aux18,aux19,aux20], 1)
watchsumleq([aux7,aux8,aux9,aux10,aux11,aux12,aux13], 1)
watchsumgeq([aux7,aux8,aux9,aux10,aux11,aux12,aux13], 1)
product(bibd[1, 0],bibd[2, 0], aux42)
product(bibd[1, 1],bibd[2, 1], aux43)
product(bibd[1, 2],bibd[2, 2], aux44)
product(bibd[1, 3],bibd[2, 3], aux45)
product(bibd[1, 4],bibd[2, 4], aux46)
product(bibd[1, 5],bibd[2, 5], aux47)
product(bibd[1, 6],bibd[2, 6], aux48)
product(bibd[1, 0],bibd[3, 0], aux49)
product(bibd[1, 1],bibd[3, 1], aux50)
product(bibd[1, 2],bibd[3, 2], aux51)
product(bibd[1, 3],bibd[3, 3], aux52)
product(bibd[1, 4],bibd[3, 4], aux53)
product(bibd[1, 5],bibd[3, 5], aux54)
product(bibd[1, 6],bibd[3, 6], aux55)
product(bibd[1, 0],bibd[4, 0], aux56)
product(bibd[1, 1],bibd[4, 1], aux57)
product(bibd[1, 2],bibd[4, 2], aux58)
product(bibd[1, 3],bibd[4, 3], aux59)
product(bibd[1, 4],bibd[4, 4], aux60)
product(bibd[1, 5],bibd[4, 5], aux61)
product(bibd[1, 6],bibd[4, 6], aux62)
product(bibd[1, 0],bibd[5, 0], aux63)
product(bibd[1, 1],bibd[5, 1], aux64)
product(bibd[1, 2],bibd[5, 2], aux65)
product(bibd[1, 3],bibd[5, 3], aux66)
product(bibd[1, 4],bibd[5, 4], aux67)
product(bibd[1, 5],bibd[5, 5], aux68)
product(bibd[1, 6],bibd[5, 6], aux69)
product(bibd[1, 0],bibd[6, 0], aux70)
product(bibd[1, 1],bibd[6, 1], aux71)
product(bibd[1, 2],bibd[6, 2], aux72)
product(bibd[1, 3],bibd[6, 3], aux73)
product(bibd[1, 4],bibd[6, 4], aux74)
product(bibd[1, 5],bibd[6, 5], aux75)
product(bibd[1, 6],bibd[6, 6], aux76)
watchsumleq([aux70,aux71,aux72,aux73,aux74,aux75,aux76], 1)
watchsumgeq([aux70,aux71,aux72,aux73,aux74,aux75,aux76], 1)
watchsumleq([aux63,aux64,aux65,aux66,aux67,aux68,aux69], 1)
watchsumgeq([aux63,aux64,aux65,aux66,aux67,aux68,aux69], 1)
watchsumleq([aux56,aux57,aux58,aux59,aux60,aux61,aux62], 1)
watchsumgeq([aux56,aux57,aux58,aux59,aux60,aux61,aux62], 1)
watchsumleq([aux49,aux50,aux51,aux52,aux53,aux54,aux55], 1)
watchsumgeq([aux49,aux50,aux51,aux52,aux53,aux54,aux55], 1)
product(bibd[2, 0],bibd[3, 0], aux77)
product(bibd[2, 1],bibd[3, 1], aux78)
product(bibd[2, 2],bibd[3, 2], aux79)
product(bibd[2, 3],bibd[3, 3], aux80)
product(bibd[2, 4],bibd[3, 4], aux81)
product(bibd[2, 5],bibd[3, 5], aux82)
product(bibd[2, 6],bibd[3, 6], aux83)
product(bibd[2, 0],bibd[4, 0], aux84)
product(bibd[2, 1],bibd[4, 1], aux85)
product(bibd[2, 2],bibd[4, 2], aux86)
product(bibd[2, 3],bibd[4, 3], aux87)
product(bibd[2, 4],bibd[4, 4], aux88)
product(bibd[2, 5],bibd[4, 5], aux89)
product(bibd[2, 6],bibd[4, 6], aux90)
product(bibd[2, 0],bibd[5, 0], aux91)
product(bibd[2, 1],bibd[5, 1], aux92)
product(bibd[2, 2],bibd[5, 2], aux93)
product(bibd[2, 3],bibd[5, 3], aux94)
product(bibd[2, 4],bibd[5, 4], aux95)
product(bibd[2, 5],bibd[5, 5], aux96)
product(bibd[2, 6],bibd[5, 6], aux97)
product(bibd[2, 0],bibd[6, 0], aux98)
product(bibd[2, 1],bibd[6, 1], aux99)
product(bibd[2, 2],bibd[6, 2], aux100)
product(bibd[2, 3],bibd[6, 3], aux101)
product(bibd[2, 4],bibd[6, 4], aux102)
product(bibd[2, 5],bibd[6, 5], aux103)
product(bibd[2, 6],bibd[6, 6], aux104)
watchsumleq([aux98,aux99,aux100,aux101,aux102,aux103,aux104], 1)
watchsumgeq([aux98,aux99,aux100,aux101,aux102,aux103,aux104], 1)
watchsumleq([aux91,aux92,aux93,aux94,aux95,aux96,aux97], 1)
watchsumgeq([aux91,aux92,aux93,aux94,aux95,aux96,aux97], 1)
watchsumleq([aux84,aux85,aux86,aux87,aux88,aux89,aux90], 1)
watchsumgeq([aux84,aux85,aux86,aux87,aux88,aux89,aux90], 1)
product(bibd[3, 0],bibd[4, 0], aux105)
product(bibd[3, 1],bibd[4, 1], aux106)
product(bibd[3, 2],bibd[4, 2], aux107)
product(bibd[3, 3],bibd[4, 3], aux108)
product(bibd[3, 4],bibd[4, 4], aux109)
product(bibd[3, 5],bibd[4, 5], aux110)
product(bibd[3, 6],bibd[4, 6], aux111)
product(bibd[3, 0],bibd[5, 0], aux112)
product(bibd[3, 1],bibd[5, 1], aux113)
product(bibd[3, 2],bibd[5, 2], aux114)
product(bibd[3, 3],bibd[5, 3], aux115)
product(bibd[3, 4],bibd[5, 4], aux116)
product(bibd[3, 5],bibd[5, 5], aux117)
product(bibd[3, 6],bibd[5, 6], aux118)
product(bibd[3, 0],bibd[6, 0], aux119)
product(bibd[3, 1],bibd[6, 1], aux120)
product(bibd[3, 2],bibd[6, 2], aux121)
product(bibd[3, 3],bibd[6, 3], aux122)
product(bibd[3, 4],bibd[6, 4], aux123)
product(bibd[3, 5],bibd[6, 5], aux124)
product(bibd[3, 6],bibd[6, 6], aux125)
watchsumleq([aux119,aux120,aux121,aux122,aux123,aux124,aux125], 1)
watchsumgeq([aux119,aux120,aux121,aux122,aux123,aux124,aux125], 1)
watchsumleq([aux112,aux113,aux114,aux115,aux116,aux117,aux118], 1)
watchsumgeq([aux112,aux113,aux114,aux115,aux116,aux117,aux118], 1)
product(bibd[4, 0],bibd[5, 0], aux126)
product(bibd[4, 1],bibd[5, 1], aux127)
product(bibd[4, 2],bibd[5, 2], aux128)
product(bibd[4, 3],bibd[5, 3], aux129)
product(bibd[4, 4],bibd[5, 4], aux130)
product(bibd[4, 5],bibd[5, 5], aux131)
product(bibd[4, 6],bibd[5, 6], aux132)
product(bibd[4, 0],bibd[6, 0], aux133)
product(bibd[4, 1],bibd[6, 1], aux134)
product(bibd[4, 2],bibd[6, 2], aux135)
product(bibd[4, 3],bibd[6, 3], aux136)
product(bibd[4, 4],bibd[6, 4], aux137)
product(bibd[4, 5],bibd[6, 5], aux138)
product(bibd[4, 6],bibd[6, 6], aux139)
watchsumleq([aux133,aux134,aux135,aux136,aux137,aux138,aux139], 1)
watchsumgeq([aux133,aux134,aux135,aux136,aux137,aux138,aux139], 1)
product(bibd[5, 0],bibd[6, 0], aux140)
product(bibd[5, 1],bibd[6, 1], aux141)
product(bibd[5, 2],bibd[6, 2], aux142)
product(bibd[5, 3],bibd[6, 3], aux143)
product(bibd[5, 4],bibd[6, 4], aux144)
product(bibd[5, 5],bibd[6, 5], aux145)
product(bibd[5, 6],bibd[6, 6], aux146)
watchsumleq([aux140,aux141,aux142,aux143,aux144,aux145,aux146], 1)
watchsumgeq([aux140,aux141,aux142,aux143,aux144,aux145,aux146], 1)
watchsumleq([aux126,aux127,aux128,aux129,aux130,aux131,aux132], 1)
watchsumgeq([aux126,aux127,aux128,aux129,aux130,aux131,aux132], 1)
watchsumleq([aux105,aux106,aux107,aux108,aux109,aux110,aux111], 1)
watchsumgeq([aux105,aux106,aux107,aux108,aux109,aux110,aux111], 1)
watchsumleq([aux77,aux78,aux79,aux80,aux81,aux82,aux83], 1)
watchsumgeq([aux77,aux78,aux79,aux80,aux81,aux82,aux83], 1)
watchsumleq([aux42,aux43,aux44,aux45,aux46,aux47,aux48], 1)
watchsumgeq([aux42,aux43,aux44,aux45,aux46,aux47,aux48], 1)
watchsumleq([aux0,aux1,aux2,aux3,aux4,aux5,aux6], 1)
watchsumgeq([aux0,aux1,aux2,aux3,aux4,aux5,aux6], 1)
watchsumleq([bibd[6, 0],bibd[6, 1],bibd[6, 2],bibd[6, 3],bibd[6, 4],bibd[6, 5],bibd[6, 6]], 3)
watchsumgeq([bibd[6, 0],bibd[6, 1],bibd[6, 2],bibd[6, 3],bibd[6, 4],bibd[6, 5],bibd[6, 6]], 3)
watchsumleq([bibd[5, 0],bibd[5, 1],bibd[5, 2],bibd[5, 3],bibd[5, 4],bibd[5, 5],bibd[5, 6]], 3)
watchsumgeq([bibd[5, 0],bibd[5, 1],bibd[5, 2],bibd[5, 3],bibd[5, 4],bibd[5, 5],bibd[5, 6]], 3)
watchsumleq([bibd[4, 0],bibd[4, 1],bibd[4, 2],bibd[4, 3],bibd[4, 4],bibd[4, 5],bibd[4, 6]], 3)
watchsumgeq([bibd[4, 0],bibd[4, 1],bibd[4, 2],bibd[4, 3],bibd[4, 4],bibd[4, 5],bibd[4, 6]], 3)
watchsumleq([bibd[3, 0],bibd[3, 1],bibd[3, 2],bibd[3, 3],bibd[3, 4],bibd[3, 5],bibd[3, 6]], 3)
watchsumgeq([bibd[3, 0],bibd[3, 1],bibd[3, 2],bibd[3, 3],bibd[3, 4],bibd[3, 5],bibd[3, 6]], 3)
watchsumleq([bibd[2, 0],bibd[2, 1],bibd[2, 2],bibd[2, 3],bibd[2, 4],bibd[2, 5],bibd[2, 6]], 3)
watchsumgeq([bibd[2, 0],bibd[2, 1],bibd[2, 2],bibd[2, 3],bibd[2, 4],bibd[2, 5],bibd[2, 6]], 3)
watchsumleq([bibd[1, 0],bibd[1, 1],bibd[1, 2],bibd[1, 3],bibd[1, 4],bibd[1, 5],bibd[1, 6]], 3)
watchsumgeq([bibd[1, 0],bibd[1, 1],bibd[1, 2],bibd[1, 3],bibd[1, 4],bibd[1, 5],bibd[1, 6]], 3)
watchsumleq([bibd[0, 0],bibd[0, 1],bibd[0, 2],bibd[0, 3],bibd[0, 4],bibd[0, 5],bibd[0, 6]], 3)
watchsumgeq([bibd[0, 0],bibd[0, 1],bibd[0, 2],bibd[0, 3],bibd[0, 4],bibd[0, 5],bibd[0, 6]], 3)
watchsumleq([bibd[0, 6],bibd[1, 6],bibd[2, 6],bibd[3, 6],bibd[4, 6],bibd[5, 6],bibd[6, 6]], 3)
watchsumgeq([bibd[0, 6],bibd[1, 6],bibd[2, 6],bibd[3, 6],bibd[4, 6],bibd[5, 6],bibd[6, 6]], 3)
watchsumleq([bibd[0, 5],bibd[1, 5],bibd[2, 5],bibd[3, 5],bibd[4, 5],bibd[5, 5],bibd[6, 5]], 3)
watchsumgeq([bibd[0, 5],bibd[1, 5],bibd[2, 5],bibd[3, 5],bibd[4, 5],bibd[5, 5],bibd[6, 5]], 3)
watchsumleq([bibd[0, 4],bibd[1, 4],bibd[2, 4],bibd[3, 4],bibd[4, 4],bibd[5, 4],bibd[6, 4]], 3)
watchsumgeq([bibd[0, 4],bibd[1, 4],bibd[2, 4],bibd[3, 4],bibd[4, 4],bibd[5, 4],bibd[6, 4]], 3)
watchsumleq([bibd[0, 3],bibd[1, 3],bibd[2, 3],bibd[3, 3],bibd[4, 3],bibd[5, 3],bibd[6, 3]], 3)
watchsumgeq([bibd[0, 3],bibd[1, 3],bibd[2, 3],bibd[3, 3],bibd[4, 3],bibd[5, 3],bibd[6, 3]], 3)
watchsumleq([bibd[0, 2],bibd[1, 2],bibd[2, 2],bibd[3, 2],bibd[4, 2],bibd[5, 2],bibd[6, 2]], 3)
watchsumgeq([bibd[0, 2],bibd[1, 2],bibd[2, 2],bibd[3, 2],bibd[4, 2],bibd[5, 2],bibd[6, 2]], 3)
watchsumleq([bibd[0, 1],bibd[1, 1],bibd[2, 1],bibd[3, 1],bibd[4, 1],bibd[5, 1],bibd[6, 1]], 3)
watchsumgeq([bibd[0, 1],bibd[1, 1],bibd[2, 1],bibd[3, 1],bibd[4, 1],bibd[5, 1],bibd[6, 1]], 3)
watchsumleq([bibd[0, 0],bibd[1, 0],bibd[2, 0],bibd[3, 0],bibd[4, 0],bibd[5, 0],bibd[6, 0]], 3)
watchsumgeq([bibd[0, 0],bibd[1, 0],bibd[2, 0],bibd[3, 0],bibd[4, 0],bibd[5, 0],bibd[6, 0]], 3)

**EOF**
